SU2_CFD/src/solvers/CTurbSolver.cpp

```diff
@@ -508,6 +511,24 @@ void CTurbSolver::BC_Fluid_Interface(CGeometry *geometry, CSolver **solver_conta
 
 }
 
+void CTurbSolver::Impose_Fixed_Values(CGeometry *geometry, CConfig *config){
+  SU2_OMP_FOR_DYN(omp_chunk_size)
+  for (unsigned long iPoint = 0; iPoint < nPointDomain; iPoint++) {
+    if (geometry->nodes->GetDomain(iPoint)) {
+      su2double* velocity_inf = config->GetVelocity_FreeStreamND();
+      su2double sp =
+        GeometryToolbox::DotProduct(nDim, geometry->nodes->GetCoord(iPoint), velocity_inf)
+        / GeometryToolbox::Norm(nDim, velocity_inf);
+      if(sp < config->GetTurb_Fixed_Values_MaxScalarProd()){
+        /*--- Set the solution values and zero the residual ---*/
+        nodes->SetSolution_Old(iPoint, Solution_Inf);
+        nodes->SetSolution(iPoint, Solution_Inf);
+        LinSysRes.SetBlock_Zero(iPoint);
```

Comment:
I think it would be better to also manipulate the Jacobian (1's in the diagonal and desired solution in the RHS, see the wall BC's of the turbulence solvers) as that also improves its condition number.

Reply:
I'll add the `Jacobian.DeleteValsRowi`